README.md

```diff
@@ -133,6 +133,22 @@ Below are the results from my home made suite of variable sized LP(s)
   monster_II: { constraints: 842, variables: 924, ints: 112, time: 0.649073406 } }
 ```
 
+What if my Mixed-Integer Problem takes too long to Solve?
+----------------------
+
+For large scale integer problems the solving process can take increasingly long. However, oftentimes the solution to these problems does not have to be the absolute best possible solution, but rather a solution relatively close to the optimal one. 
```

Comment:
This 1st sentence doesn't seem quite right (english is not my native tongue though). Maybe : "For large scale integer problems the solving process can take increasingly more time."